app/controllers/claims/schools/mentors_controller.rb

```diff
@@ -1,7 +1,26 @@
 class Claims::Schools::MentorsController < ApplicationController
   include Claims::BelongsToSchool
+  before_action :set_mentor, only: %i[show remove destroy]
 
   def index
     @pagy, @mentors = pagy(@school.mentors.order(:first_name, :last_name))
   end
+
+  def show; end
+
+  def remove; end
+
+  def destroy
+    mentor_membership = @mentor.mentor_memberships.find_by(school: @school)
+    mentor_membership.destroy!
+
+    redirect_to claims_school_mentors_path(@school)
+    flash[:success] = t(".mentor_removed")
```

Comment:
I've been doing things _slightly_ different here. I wonder what your thoughts are?

```suggestion
    redirect_to claims_school_mentors_path(@school), flash: { success: t(".success") }
```

1. Inline the flash as it acts like a modifier to the redirection, not logically a step to perform _after_ redirection.
2. Use the `success` key as a behavioural identifier, rather than direct translation. The expanded form is `claims.schools.mentors.destroy.success` which provides the context required for the `.success` key.